cmd/agent/main.go

```diff
@@ -1,3 +1,35 @@
 package main
 
-func main() {}
+import (
+	"evgen3000/go-musthave-metrics-tpl.git/cmd/agent/client"
+	"flag"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	hostEnv, isHostEnv := os.LookupEnv("ADDRESS")
+	reportIntervalEnv, isReportIntervalEn := os.LookupEnv("REPORT_INTERVAL")
+	pollIntervalEnv, isPollIntervalEnv := os.LookupEnv("POLL_INTERVAL")
+
+	hostFlag := flag.String("a", "localhost:8080", "Host IP address and port.")
```

Comment:
[SUGGESTION] You can move it to the `flag.go` file.

Reply:
sure, I've the same opinion. But idk, how to split same code from agend and server for different folders/packeges.

Reply:
https://github.com/golang-standards/project-layout/tree/master/internal/app/_your_app_